test-utils/src/main/java/org/hibernate/validator/testutil/DescriptorAssert.java

```diff
@@ -34,10 +34,13 @@ public static GroupConversionDescriptorSetAssert assertThat(Set<GroupConversionD
 	 *
 	 * @author Gunnar Morling
 	 */
-	public static class GroupConversionDescriptorSetAssert extends CollectionAssert {
+	public static class GroupConversionDescriptorSetAssert extends IterableAssert {
```

Comment:
IterableAssert accepts a generic parameter. If you use it, you don't have to keep the `actual` parameter in a specific variable. I fixed it.